extensions/README.md

```diff
@@ -140,3 +130,74 @@ Since it's added to the beta array, you need to load the beta blocks as explaine
 
 `wp jetpack scaffold block "Jukebox" --keywords="music, audio, media"`
 
+## Good to know when developing Gutenberg extensions
+
+## The Build
+
+- Compiled extensions are output to `_inc/blocks`
+- You can view the various build commands in `package.json`
+- You can see the build configuration in `webpack.config.extensions.js`
+
+If you need to modify the build process, bear in mind that config files are also
+synced to WordPress.com via Fusion. Consult with a Jetpack crew member to ensure
+you test the new build in both environments.
+
+## Debugging
+
+Setting these might be useful for debugging with block editor:
+
+```php
+define( 'SCRIPT_DEBUG', true );
+define( 'GUTENBERG_DEVELOPMENT_MODE', true );
+```
+
+You could add these to `docker/wordpress/wp-config.php` in your Docker environment.
+
+[G Debugger](https://wordpress.org/plugins/g-debugger/) plugin might come handy, too.
+
+### Don't worry about dependencies
+
+The build takes care of core dependencies for both editor and view -scripts. React, lodash and `@wordpress/*` [dependencies](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md) are externalised and automatically enqueued in PHP for your extension.
+
+Extensions _always_ get [Gutenberg's polyfill scripts](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md#polyfill-scripts) enqueued so you can safely use methods not supported by older browsers such as IE11.
+
+### Jetpack sidebar
```

Comment:
I don't know if "Jetpack sidebar" means much for someone who is just starting to contribute to Jetpack. Maybe we could add a screenshot here, or a little bit of explanation about the differences between blocks and plugins?

Reply:
I left a screenshot out as those are pain to maintain but elaborated more what it is and how to find it in https://github.com/Automattic/jetpack/pull/12050/commits/22c823184c9bc4f5b44ad8c1a2ea32a229d0c656